rabbitmq/rabbitClient.go

```diff
@@ -39,6 +39,19 @@ func NewRabbitMQClient(url string) (*rabbitMqClient, error) {
 	return rc, nil
 }
 
+// ExchangeDeclare will declare an exchange
+func (rc *rabbitMqClient) ExchangeDeclare(name, kind string) error {
+	return rc.ch.ExchangeDeclare(
+		name,  // name
+		kind,  // type
+		true,  // durable
```

Comment:
please remove the comments and add variables for L47-L51

Reply:
done